README.md

```diff
@@ -45,15 +45,20 @@ For any improvements on the website, we should continuously measure its impact o
 
 ## Getting Started
 
-First, install dependencies and fetch data to your local machine
+First, clone the repository
```

Comment:
```suggestion
First, clone the repository.

```

Nitpick: double-line separator for proper paragraphs, and period at the end of sentence.